### ISSUE TITLE
Exception on Safari

### ISSUE THREAD

**Issue by sbdchd** [edgedb/edgedb-studio#34]
This bit of code is exploding:

https://github.com/edgedb/edgedb-studio/blob/main/shared/studio/tabs/schema/state/textView.ts#L246-L248

The value of `schemaData.scalars.values()` is 2k lines so put it in a gist:

https://gist.github.com/sbdchd/1b1f50612d6e66f49e56502850739c76

but here's a screenshot

<img width="719" alt="image" src="https://user-images.githubusercontent.com/7340772/181672113-3a393d2c-b71b-4188-ad28-3c4bb0413f01.png">

and the errors:

<img width="747" alt="image" src="https://user-images.githubusercontent.com/7340772/181672898-eaa31fd8-f63e-4d52-a164-123e123262b0.png">

```
[mobx] uncaught error in 'Reaction[Autorun]'
TypeError: undefined is not an object (evaluating 'e.name.localeCompare')

Unhandled Promise Rejection: TypeError: undefined is not an object (evaluating 'e.name.localeCompare')
```

**Comment by 1st1** [comment]
Thanks, we'll take a look!

**Comment by jaclarke** [comment]
@sbdchd Can you post your schema? Also did this error happen the first time you used the UI, or did the error happen after a page reload?

**Comment by sbdchd** [comment]
I don't think it initially happened. I'm pretty sure I navigated between a couple projects and then I tried to type the REPL and I couldn't type.

Then I looked in the console and saw an error. Later when navigating more stuff started breaking more as in white screen.

I have 3 projects setup in the UI, one is the example the default schema, and a couple empty projects with:

```
module default{};
```

for the sdl
